src/components/dialog/dialog.scss

```diff
@@ -16,7 +17,7 @@
  * @prop --dialog-max-width: Max width of the dialog.
  * @prop --dialog-max-height: Max height of the dialog.
  * @prop --dialog-border-radius: Border radius of the dialog corners
- * @prop --dialog-padding-top-bottom: Padding on top and bottom of dialog content. Defaults to `1.5rem`.
+ * @prop --dialog-padding-top-bottom: Padding on top and bottom of dialog content. Affects the height of fade-out effects on top and bottom edges when the content is scrollable and has overflowed out of the content area. Defaults to `1.5rem`. Note that if you use this variable and set it to numbers smaller than 1rem, you will loose the fade-out effects on the edges. If you have set these paddings to `0`, losing the fade out effects should be however fine for your use case! Because in such a case your intention is to handle the `overflow` internally in the component that is displayed in the dialog's content.
```

Comment:
it might be hard to know what these docs are trying to say. but this is the background story of adding those variables: https://github.com/Lundalogik/lime-elements/issues/1953 some appers user the whole dialog surface to render their "fullscreen apps", and they don't want our paddings. They just set it to zero: https://github.com/search?q=org%3ALundalogik%20--dialog-padding-top-bottom&type=code